backend/pyproject.toml

```diff
@@ -44,5 +42,7 @@ httpx = "^0.23.3"
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
-[tool.isort]
-profile = "black"
+[tool.ruff]
+ignore = ["E501"]
+exclude = ["alembic/versions/*"]
+extend-select = ["I", "SIM", "W", "UP", "TCH"]
```

Comment:
I = isort
F is on by default and stands for flake see https://beta.ruff.rs/docs/rules/#pyflakes-f